README.md

```diff
@@ -2,6 +2,35 @@
 
 This is the package bundle you can install alongside a plain Neos to play around with the event-sourced CR.
 
+## Feature comparison
+
+✅ Done
+
+⏩ Currently worked on
+
+🚫 Will not be supported
+
+| Feature                     | Current CR | Event Sourced CR |
+| --------------------------- |:----------:|:----------------:|
+| **Basics**                    |            |                  |
+| Create/ Edit / Delete Nodes |     ✅     |        ⏩        |
+| Query Nodes                 |     ✅     |        ⏩        |
+| Cut / Copy / Paste          |     ✅     |                  |
+| Move Nodes                  |     ✅     |                  |
```

Comment:
I think this is supported as soon as we get to a working state again and rendering works :)